scriptmodules/ports/ut.sh

```diff
@@ -0,0 +1,140 @@
+}
+
+function install_bin_ut() {
+    local version="$(_get_branch_ut)"
+    local arch="$(dpkg --print-architecture)"
+
+    # For some reason, it failed when using "$rp_module_repo", this works perfectly.
+    local base_url="https://github.com/OldUnreal/UnrealTournamentPatches"
+    local dl_file="OldUnreal-UTPatch${version}-Linux-${arch}.tar.bz2"
+    local dl_url="${base_url}/releases/download/v${version}/${dl_file}"
+
+    # The download files use "x86" for the i386 architecture
+    [[ "${arch}" == "i386" ]] && arch="x86"
+
+    downloadAndExtract "$dl_url" "$md_inst" "--no-same-owner"
+}
+
+function __config_game_data() {
+    local ut_game_dir=$1
+
+    if [[ ! -d "$romdir/ports/ut/$ut_game_dir" ]]; then
+        mkdir -p "$romdir/ports/ut/$ut_game_dir"
+        chown -R "$__user":"$__group" "$romdir/ports/ut/$ut_game_dir"
+    else
+        chown "$__user":"$__group" "$romdir/ports/ut/$ut_game_dir"
+    fi
+
+    if [[ -d "$md_inst/$ut_game_dir" ]]; then
+        cd "$md_inst/$ut_game_dir"
+        for file in $(ls -d *); do
+
+            echo "Moving $md_inst/$ut_game_dir/$file -> $romdir/ports/$ut_game_dir/$file"
+
+            if [[ -d "$md_inst/$ut_game_dir/$file" ]]; then
+                if [[ ! -d "$romdir/ports/ut/$ut_game_dir/$file" ]]; then
+                    mv "$md_inst/$ut_game_dir/$file" "$romdir/ports/ut/$ut_game_dir/$file"
+                else
+                    rm -rf "$romdir/ports/ut/$ut_game_dir/$file"
+                    mv "$md_inst/$ut_game_dir/$file" "$romdir/ports/ut/$ut_game_dir/$file"
+                fi
+            else
+                mv "$md_inst/$ut_game_dir/$file" "$romdir/ports/ut/$ut_game_dir/$file"
+            fi
+        done
+
+        rm -rf "$md_inst/$ut_game_dir"
+    fi
+
+    ln -snf "$romdir/ports/ut/$ut_game_dir" "$md_inst/$ut_game_dir"
+}
+
+function game_data_ut() {
+
+    for dir in Help Maps Music Sounds Textures Web; do
+
+        # Ensure we aren't moving files that are already in place.
+        # Eliminates 'mv: '$src/$file' and '$dst/$file' are the same file' errors.
+        if [[ ! -h "$md_inst/$dir" ]]; then
+            __config_game_data "$dir"
+        fi
+    done
+
+    local bonus_pack_4_url="https://unreal-archive-files-s3.s3.us-west-002.backblazeb2.com/patches-updates/Unreal%20Tournament/Bonus%20Packs/utbonuspack4-zip.zip"
+    downloadAndExtract "$bonus_pack_4_url" "$romdir/ports/ut/"
+
+    chown -R "$__user":"$__group" "$romdir/ports/ut"
+    find  "$romdir/ports/ut" -type f -exec chmod 644 {} \;
+    find  "$romdir/ports/ut" -type d -exec chmod 755 {} \;
+
+    # We only want to install this if it is not already installed.
+    if [[ ! -f "$home/.utpg/UnrealTournament.ini" ]]; then
+        cp "$md_data/UnrealTournament.ini" "$home/.utpg/UnrealTournament.ini"
+        chown "$__user":"$__group" "$home/.utpg/UnrealTournament.ini"
+        chmod 644 "$home/.utpg/UnrealTournament.ini"
+    fi
+}
+
+
+function configure_ut() {
+    if isPlatform "x86"; then
```

Comment:
so doing a clean install let me get a error. 
we need to move 
```
    moveConfigDir "$home/.utpg" "$md_conf_root/ut"
```
to the first line under the configure function or a error will come up and the install will have to be ran again for it to complete correctly

Reply:
It doesn't need to go to the beginning, but it does need to happen before we attempt to install 'UnrealTournament.ini'

I moved that code until the very end.